CSS/style.css

```diff
@@ -0,0 +1,206 @@
+#blue {
+    background-color: rgb(4, 4, 90);
+    height: 105px;
+    display: flex;
+}
+#gray {
+    font-family: Verdana, sans-serif;
+    text-align: center;
+    margin: auto;
+    width: 28%;
+    height: 46px;
+    background-color: white;
+    border: 1.5px solid red;
+}
+#bluebox {
+    margin: 0%;
+    padding: 0%;
+    font-size: 20px;
+}
+#graybox {
```

Comment:
Try to use proper naming convention. Here graybox and bluebox does depicts section it represents

Reply:
Naming conversations are changed.
<img width="270" alt="Screenshot 2023-02-14 105753" src="https://user-images.githubusercontent.com/124386275/218647594-ea6f3e81-3ec6-438b-8d4d-12d5ae5f5adf.png">